packages/leaders.lua

```diff
@@ -65,12 +65,12 @@ allow you to define your own leaders. For example:
 
 \begin{verbatim}
 \line
-A \\leaders[width=30pt]\{\\font[features="+ornm"]{iI}\} B
+A \\leaders[width=40pt]\{/\\\} B
```

Comment:
This needs another escaped backslash I believe:

```suggestion
A \\leaders[width=40pt]\{/\\\\\\} B
```

Dizzying, but to show in the output that you backslash is escaped you need two pairs, plus one more to escape the closing brace to get an actual brace.